### PR TITLE
Update the title, description and readme for the Legacy OmniSharp Extension

### PR DESCRIPTION
**Pull request by DustinCampbell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-To **run and develop** do the following:
+In the meantime, you can install and use this the Legacy OmniSharp extension to enjoy a great C# editing experience on OS X and Linux for .csproj-based projects, such as Unity and Xamarin.
 
-* run `npm i`
-* open in VS Code
-* *(opt)* run `tsc -w`, make code changes
-* hit F5
+**Please note the following important details before installing this extension:**
 
+1. You should only use this extension on OS X and Linux if you are developing on .csproj projects. The C# extension officially supports .csproj projects on Windows. Also, if you are doing .NET COre development on Windows, OS X or Linux, the C# extension is the appropriate choice.
```
Comment (staceyhaffner) [inline README.md]: 'o' in Core is incorrectly capitalized.

Comment (DustinCampbell) [inline README.md]: Already fixed -- thanks!

**Comment by gregg-miskelly** [comment]
@DustinCampbell would it make sense to copy over the instructions for getting ASP.NET 5 Core RC1 onto the box since it is pretty tricky to dig up those instructions these days?

**Comment by DustinCampbell** [comment]
Good idea. I'll add that.

**Pull request merged**